packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -210,9 +253,19 @@ const Tab = ({
             <div className={variant === 'framework' ? cy('inner', 'draggable-inner') : cx('inner')}>
               <IconKnurling />
               <VisuallyHiddenText aria-hidden id={responseId} text={onFocusResponse} />
-              {customDisplay || icon}
+              <div className={cy('draggable-icon')}>{customDisplay || icon}</div>
               {(!customDisplay && !isIconOnly) && <span className={variant === 'framework' ? cy('label') : cx('label')}>{label}</span>}
             </div>
+            {isClosable && (
+            <div
+              className={cx('tabs-remove-button')}
+              type="button"
+              aria-label={tabDeleteLabel}
+              onClick={onCloseClick}
+            >
+              <IconClose a11yLabel="Close Button" />
```

Comment:
```suggestion
              <IconClose a11yLabel="Close Button" />
```
needs translations for string "Close Button" !!

Reply:
This Icon as added for decorative icon, which wont get focus hence removing the a11ylabel